config/server.properties

```diff
@@ -12,7 +12,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# see kafka.server.KafkaConfig for additional details and defaults
```

Comment:
I think this line is intended, although we should probably introduce an empty line between the license text and this comment.